fiftyone/core/expressions.py

```diff
@@ -1859,6 +1859,19 @@ def if_else(self, true_expr, false_expr):
             {"$cond": {"if": self, "then": true_expr, "else": false_expr}}
         )
 
+    def if_null(self, false_expr):
+        """Returns either this expression or ``false_expr`` if this expression is null.
+        This is a shortcut for ``if_else(self.is_null(), false_expr, self)`` and is useful
+        for replacing null values in a field with a default value.
+
+        Args:
```

Comment:
By convention (look at all other methods in this module), all `ViewExpression` methods should contain an example usage in their docstring under an `Examples::` block. Can you please add this?